src/utils.js

```diff
@@ -190,14 +190,21 @@ export const handleConfigLogin = (token, lastProjectId) => {
 }
 
 export const onAppLoad = () => {
-  client.fetchEnv().then(({isProd, version}) => {
+  client.fetchEnv().then(({ isProd, version }) => {
     const token = localStorage.getItem("token")
     localStorage.getItem("isProd", isProd.toString())
     store.dispatch(set("version", version))
     if (isProd && !token) {
       history.push("/mission-control/login")
       return
     }
+    if (isProd && token) {
+      client.refreshToken(token)
+        .then(token => {
+          localStorage.setItem("token", token);
+          handleConfigLogin(token)
+        })
+    }
 
     let lastProjectId = null
     const urlParams = window.location.pathname.split("/")
```

Comment:
Put this logic before the if of `client.refreshToken` so that you can pass `lastProjectId` in the `handleConfigLogin` in the success of `client.refreshToken`